### DISCUSSION TITLE
Should we upgrade Elsa 1 to 2 or wait for v3?

### DISCUSSION THREAD

**Discussion by xitac** [elsa-workflows/elsa-core#3295] [General]
Hello,
We've been using Elsa workflow version 1 in our application and its working fine for what we need, but it's a realy a minor use case.
Currently we are planning some heavier use cases of workflows and since we re not utilizing elsa that much right now, we would definitely switch to a newer version .. so I'm wondering if we should switch to Elsa v2 or maybe wait for Elsa v3? How long would be the wait? :)

**Comment by sfmskywalker** [comment]
If you can wait for 3 or 4 more months, I’d definitely wait for V3 :)

You’ll be more used to the designer, the runtime is vastly superior, as is the extensibility, programming model, and not to mention overall quality.

**Reply by xitac** [reply]
Thank you for the quick reply and for great work. I guess we'll be waiting :)
